cmd/cloudshell_open/main.go

```diff
@@ -308,6 +309,20 @@ func run(opts runOpts) error {
 		}
 	}
 
+	hookEnvs := append(envs,
```

Comment:
so there's a bug here:

if you do `append(envs, something...)`, it actually overwrites `env` slice as well. the underlying slice is modified.

I recommend doing this instead

```
foo = append([]string{v1, v2, v3}, envs...)
```